yax/modules/identify_informative_hits.py

```diff
@@ -0,0 +1,217 @@
+from yax.shared.utilities import taxidtool
+from yax.artifacts import TaxIDData, InformativeAlignmentData, IdentifiedTaxa
+from yax.state.type.parameters import Int, File
+
+
+def main(working_dir, output, details, identified_taxa_art: IdentifiedTaxa,
+         tax_id_data_art: TaxIDData, lca_dist: Int, min_hits: Int,
+         min_relative_hits: Int, sequences_input_fp: File) \
+            -> InformativeAlignmentData:
```

Comment:
it has a tree, and most of the stuff it does actually, because I don't want to call the writing of files directly from outside the artifact and I don't know another way to do this than to make the artifact have a concept of these things.